test/acceptance/cli-test.js

```diff
@@ -142,7 +142,10 @@ describe('ember-template-lint executable', function() {
           '  1:4  error  Non-translated string used  no-bare-strings',
           '  2:5  error  Non-translated string used  no-bare-strings',
           '',
-          '✖ 2 problems (2 errors, 0 warnings)',
+          path.resolve('./test/fixtures/with-errors/app/templates/components/foo.hbs'),
+          "  1:2  error  Ambiguous path 'fooData' is not allowed. Use '@fooData' if it is a named argument or 'this.fooData' if it is a property on 'this'. If it is a helper or component that has no arguments you must manually add it to the 'no-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['fooData'] }.  no-implicit-this",
+          '',
+          '✖ 3 problems (3 errors, 0 warnings)',
```

Comment:
This additional error seems odd, is this the one added for override support?

I'd rather keep the tests here focused on the command line itself. This would mean adding some tests here for what the warning output looks like in the command line (both `--json` version and regular), then put the tests for merging `overrides` over in `getConfigForFile` (in the `get-config-test.js`). If we give the right configuration to rules, we know they will work.